.github/workflows/build-push.yml

```diff
@@ -65,6 +65,18 @@ jobs:
         run: |
           python3 -m pip install cibuildwheel==2.22.0
 
+      - name: Install cython
+        run: |
+          python -m pip install Cython
+
+      - name: Install setuptools
+        run: |
+          python -m pip install setuptools --upgrade
+
+      - name: Install wheel
```

Comment:
Why are you adding those ? they have zero affect inside cibuildwheel that works inside docker (and it's own venv per python)

Reply:
It is still in progress, I am working on to find out why it does not recognize cython module for python >=3.12